src/utils.ts

```diff
@@ -80,7 +80,7 @@ export function addUrlQuery(url: string, query: Record<string, unknown>): string
  * @throws - Will throw if the int does not fit in 64 bits.
  * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/asUintN | MDN Demo}
  */
-export function checkUint64(int: bigint) {
+export function checkUint64(int: bigint): void {
```

Comment:
in javascript (at least) we call this type of type guarding functions assertions, can you rename it to `assertUint64` ?